code/game/objects/structures/tables_racks.dm

```diff
@@ -393,6 +399,7 @@
 			add_logs(user, src, "climbed onto")
 			user.Stun(2)
 			return 1
+			tableclimber = null
 	return 0

```

Comment:
Delete the 'tableclimber = null' line inside the If block and add it right above the return 0. This will ensure tableclimber is never not null after the proc is done running. You don't need two lines setting tableclimber to null.